includes/polling/os/ibm-dpi.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$version = end(explode(' ', trim(snmp_get($device, "sysDescr.0", "-OQv", "SNMPv2-MIB"))));
```

Comment:
We already have sysDescr value in `$device['sysDescr']`

Reply:
Mine are rebranded Lenovo PDUs actually. Maybe they're all the same now?

No MIB files that I can find. I've emailed IBM/Lenovo and we'll see if there are even any.